airflow/hooks/fs_hooks/sftp.py

```diff
@@ -0,0 +1,117 @@
+# -*- coding: utf-8 -*-
```

Comment:
Again, this shouldn't be a new hook, it should be part of the existing sftp hook.

Reply:
I agree. I believe @NielsZeilemaker enriched the existing SFTP hook recently.